website/app/components/doc/component-api/index.hbs

```diff
@@ -0,0 +1,13 @@
+<table class="doc-markdown-table doc-component-api" ...attributes>
```

Comment:
since we have not yet defined what the actual output is (a table? a list? other?) it's OK to use `doc-markdown-***` classes here, but later we have to avoid using them for styling custom components, so we can separate what is actual markdown (generated from the markdown to HTML converter) from what is a custom component (for which we have total control).